clustering.Rmd

```diff
@@ -151,30 +154,31 @@ penguin_data <- read_csv("data/penguins_standardized.csv")
 penguin_data
 ```
 
-Next, we can create a scatter plot using this data set to see if we can use
-data visualization to detect subtypes or groups in our data set.
 
-```{r 10-toy-example-plot, warning = FALSE, fig.height = 4, fig.width = 4.35, fig.cap = "Subset data of penguin standardized bill length versus standardized flipper length."}
+Next, we can create a scatter plot using this data set 
+to see if we can detect subtypes or groups in our data set.
+
+```{r 10-toy-example-plot, warning = FALSE, fig.height = 4, fig.width = 4.35, fig.cap = "Scatter plot of standardized bill length versus standardized flipper length."}
 ggplot(data, aes(x = flipper_length_standardized, y = bill_length_standardized)) +
   geom_point() +
   xlab("Flipper Length (standardized)") +
   ylab("Bill Length (standardized)")
 ```
 
-Based \index{ggplot}\index{ggplot!geom\_point} on the visualization in Figure \@ref(fig:10-toy-example-plot), we might
-suspect there are a few subtypes of penguins 
-within the observations we have in our data set.
-We can see ~ 3 groups of observations in Figure \@ref(fig:10-toy-example-plot),
+Based \index{ggplot}\index{ggplot!geom\_point} on the visualization 
+in Figure \@ref(fig:10-toy-example-plot), 
+we might suspect there are a few subtypes of penguins within our data set.
+We can see roughly 3 groups of observations in Figure \@ref(fig:10-toy-example-plot),
 including:
 
 1. a small flipper and bill length group,
 2. a small flipper length, but large bill length group, and
 3. a large  flipper and bill length group.
 
-This is interesting that we can use data visualization to uncover such patterns,
+It is interesting that we can use data visualization to uncover such patterns,
```

Comment:
This sentence isn't grammatically correct

Let's go with

```
Data visualization is a great tool to give us a rough sense for such patterns
when we have a small number of variables. But if we are to group
data&mdash;and select the number of groups&mdash;as part of 
a reproducible analysis, we need something a bit more automated.
````

Reply:
And then I am going to add this point after that as well: 

```
Additionally, finding groups through data visualization becomes more difficult 
as we increase the number of variables we consider when clustering.
```